### PR TITLE
Migrate report_webkit_barcode to 8.0

### PR DESCRIPTION
**Pull request by ovnicraft**
Minimal change in description about filter use to jinja2 

`${ helper.barcode(value) | safe}`

### CODE REVIEW

File: report_webkit_barcode/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (pedrobaeza) [inline report_webkit_barcode/__init__.py]: Please use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: report_webkit_barcode/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    'name': 'Webkit Report Barcode',
+    'version': '0.1',
+    'author': "Savoir-faire Linux,Odoo Community Association (OCA)",
+    'maintainer': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'license': 'AGPL-3',
+    'category': 'Reports/Webkit',
+    'summary': 'Adds a barcode helper to webkit reports',
+    'description': """
+Add barcodes in webkit reports
```
Comment (pedrobaeza) [inline report_webkit_barcode/__openerp__.py]: Extract the description to a README.rst file according https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: report_webkit_barcode/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Webkit Report Barcode',
+    'version': '0.1',
```
Comment (pedrobaeza) [inline report_webkit_barcode/__openerp__.py]: 8.0.1.0.0 (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)

File: report_webkit_barcode/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+installation.
+
+Contributors
+------------
+* Vincent Vinet (vincent.vinet@savoirfairelinux.com)
+""",
+    'depends': [
+        'report_webkit',
+    ],
+    'external_dependencies': {
```
Comment (pedrobaeza) [inline report_webkit_barcode/__openerp__.py]: Remove empty keys

File: report_webkit_barcode/__init__.py (hunk 1 above)
Comment (ovnicraft) [inline report_webkit_barcode/__init__.py]: @pedrobaeza this apply to `__openerp__.py` too ?

Comment (pedrobaeza) [inline report_webkit_barcode/__init__.py]: Yes

**Comment by ovnicraft** [comment]
@pedrobaeza done

File: report_webkit_barcode/README.rst [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+To embed a barcode image in a webkit report, use ``${helper.barcode(value)}``.
+Depending on template filtering, you might need to force not to use any filter
+    by doing ``${helper.barcode(value) | safe}``
+
+The function is defined as:
+  barcode(value, code='Code128', drawOpts=None, htmlAttrs=None)
+
+Parameters
+---------
```
Comment (pedrobaeza) [inline report_webkit_barcode/README.rst]: The line should have the same length as the title

File: report_webkit_barcode/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2014 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Webkit Report Barcode',
+    'version': '0.1',
```
Comment (pedrobaeza) [inline report_webkit_barcode/__openerp__.py]: 8.0.1.0.0

File: report_webkit_barcode/helper_barcode.py [hunk 7]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline report_webkit_barcode/helper_barcode.py:1]: Short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Comment (ovnicraft) [inline report_webkit_barcode/helper_barcode.py:1]: Its like template

``` python
# -*- coding: utf-8 -*-
# © <YEAR(S)> <AUTHOR(S)>
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment (pedrobaeza) [inline report_webkit_barcode/helper_barcode.py:1]: Yes, but I don't see this template here.

Comment (ovnicraft) [inline report_webkit_barcode/helper_barcode.py:1]: ok, its done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by ovnicraft** [comment]
@pedrobaeza anything else to merge ?

**Comment by pedrobaeza** [comment]
We need at least 2 more reviewers or 5 days to merge and another reviewer.

**Comment by ovnicraft** [comment]
@pedrobaeza maybe OT but would be great add a hook to post a comment in each new PR, there is any integration about this ?

**Comment by pedrobaeza** [comment]
I don't get you.

**Comment by ovnicraft** [comment]
any news about this ?

**Comment by gurneyalex** [comment]
:+1:

**Comment by gurneyalex** [comment]
@ovnicraft we are very scarce on reviewing resources in the OCA. I suggest that for any PR you make you try to review / test a couple of open PR by others. This is the way to helping to get a smoother flow.

**Pull request merged**